### ISSUE TITLE
GUI Bug: Debug Log Error for Copy Address Functionality

### ISSUE THREAD

**Issue by Rock-N-Troll** [Veil-Project/veil#541]
![image](https://user-images.githubusercontent.com/34344520/56689311-cb213200-66a8-11e9-8e4e-cf63413da720.png)

Occurred when using the right click copy functionality in the address book

**Comment by MatWaller** [comment]
Seems to be another Windows specific bug, working as intended on;

```
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.1 LTS
Release:	18.04
Codename:	bionic
```

**Comment by veilgets** [comment]
Bounty: 187.5

**Comment by veilgets** [comment]
Developer: CaveSpectre11

**Comment by CaveSpectre11** [comment]
Ready for QA/Code Review - PR #651

**Comment by codeofalltrades** [comment]
closed with pull #651
